File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('documents/UseCaseFinal.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(function(err) {
    console.error(err);
});

File: read_pdf.py
import sys
try:
    from pypdf import PdfReader
except ImportError:
    print("pypdf not installed")
    sys.exit(1)

reader = PdfReader("documents/UseCaseFinal.pdf")
text = ""
for page in reader.pages:
    text += page.extract_text() + "\n"

with open("pdf_contents.txt", "w", encoding="utf-8") as f:
    f.write(text)
print("Done")
